packages/grid/_modules_/grid/hooks/features/export/seralizers/csvSeraliser.ts

```diff
@@ -1,8 +1,18 @@
 import { GridColumns, GridRowId, GridRowModel } from '../../../../models';
 
-export function buildRow(row: GridRowModel, columns: GridColumns): Array<GridRowModel> {
-  const mappedRow: GridRowModel[] = [];
-  columns.forEach((column) => column.field !== '__check__' && mappedRow.push(row[column.field]));
+const formatCellValue = (value) => {
```

Comment:
we could name these 2 func, `serialiseRow, serialiseCell`

Reply:
done